api.json

```diff
@@ -0,0 +1,761 @@
+{
+  "title": "twine-api",
+  "version": "0.1",
+  "routes": {
+    "/sectors": {
+      "description": "",
+      "/": {
+        "GET": {
+          "description": "Retrieve all users accessible with the given scopes",
+          "auth": {},
+          "scope": {},
+          "query": {},
+          "response": {}
+        }
+      },
+      "/:id/organisations": {
+        "GET": {
+          "description": "",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList":[{
+              "id": "Number",
+              "name": "String",
+              "logo_url": "String",
+              "address1": "String",
+              "address2": "String",
+              "townCity": "String",
+              "postCode": "String",
+              "coordinates": "String",
+              "region": "String",
+              "sector": "String",
+              "registeredAt": "String",
+              "lastActiveAt": "String",
+              "modifiedAt": "String",
+              "turnoverBand": "String",
+              "360GivingId": "String",
+              "fundingProgramme": "String",
+              "fundingBody": "String",
+              "parentOrganisation": "String",
+              "childOrganisations": ["String"],
+              "subscription": "String",
+              "trainingSessions": [{
+                "trainingSessionAttendees": ["Number"],
+                "trainingSessionNotes": "String",
+                "eventDate": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/regions": {
+      "description": "",
+      "/:id/organisations": {
+        "GET": {
+          "description": "",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList":[{
```

Comment:
An idea to make the job of documenting requests/responses a little easier: define some types in the top-level of the JSON file (i.e. say what an "organisation" JSON-response looks like), and then simply reference that in the route definitions. For example:
```json
{
  "types": {
    "organisation": {
      "id": "Number",
      "name": "String",
    }
  },
  "routes": {
    "/regions": {
      "/:id/organisations": {
        "response": {
          "data": ["types.organisation"]
        },
      }
    }
  }
}
```